docs/en-US/what-is-vvv.md

```diff
@@ -0,0 +1,39 @@
+# What is VVV?
+
+Varying Vagrant Vagrants is an open source [Vagrant](https://www.vagrantup.com) configuration focused on [WordPress](https://wordpress.org) development. VVV is [MIT Licensed](https://github.com/varying-vagrant-vagrants/vvv/blob/master/LICENSE).
+
+### The Purpose of Varying Vagrant Vagrants
+
+The primary goal of Varying Vagrant Vagrants (VVV) is to provide an approachable development environment with a modern server configuration.
+
+VVV is ideal for developing themes and plugins as well as for [contributing to WordPress core](https://make.wordpress.org/core/).
+
+### What do you get?
+
+A bunch of stuff!
+
+1. [Ubuntu](http://www.ubuntu.com/) 14.04 LTS (Trusty Tahr)
+1. [WordPress Develop](https://develop.svn.wordpress.org/trunk/)
+1. [WordPress Stable](https://wordpress.org/)
+1. [WP-CLI](http://wp-cli.org/) (master branch)
+1. [nginx](http://nginx.org/) ([mainline](http://nginx.com/blog/nginx-1-6-1-7-released/) version)
+1. [mysql](https://www.mysql.com/) 5.5.x
```

Comment:
Fixed in https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1112/commits/d98cc1a937ecc199d73da03b018a79132137b56f